### ISSUE TITLE
small target detection

### ISSUE THREAD

**Issue by Yangget** [RockyXu66/Faster_RCNN_for_Open_Images_Dataset_Keras#12]
What parameters of "faster rcnn" need to be adjusted when implementing small target detection?

**Comment by RockyXu66** [comment]
That's really a good question and an interesting topic. I haven't tried. But I think change to smaller **anchor_size** might help.

**Comment by Yangget** [comment]
> 这真是一个很好的问题，也是一个有趣的话题。我没试过。但我认为改变较小的**anchor_size**可能有所帮助。

Thank you for your reply, I adjusted the anchor_size, and other parameters did not get very good performance, and compared to retinanet, the latter performed better and got 0.5 accuracy.
Would you like to help us solve this problem? I am very willing to work with you to complete this project (Super Planet Recognition). If you are willing to participate, please contact me at yzhcode@gmail.com. Thank you for your response.

**Comment by Yangget** [comment]
> That's really a good question and an interesting topic. I haven't tried. But I think change to smaller **anchor_size** might help.


Thank you for your reply, I adjusted the anchor_size, and other parameters did not get very good performance, and compared to retinanet, the latter performed better and got 0.5 accuracy.
Would you like to help us solve this problem? I am very willing to work with you to complete this project (Super Planet Recognition). If you are willing to participate, please contact me at yzhcode@gmail.com. Thank you for your response.

**Comment by Yangget** [comment]
Thank you for your reply, I adjusted the anchor_size, and other parameters
did not get very good performance, and compared to retinanet, the latter
performed better and got 0.5 accuracy. Would you like to help us solve this
problem? I am very willing to work with you to complete this project (Super
Planet Recognition). If you are willing to participate, please contact me
at yzhcode@gmail.com. Thank you for your response.

RockyXu66 <notifications@github.com> 于2019年3月27日周三 下午4:13写道：

**Comment by RockyXu66** [comment]
I just started to explore RetinaNet these days. According to the paper, I think RetinaNet achieves better performance than faster-rcnn. Why don't you directly use RetinaNet as your detector? And could you tell me more about the "Super Planet Recognition" project?

**Comment by Yangget** [comment]
> I just started to explore RetinaNet these days. According to the paper, I think RetinaNet achieves better performance than faster-rcnn. Why don't you directly use RetinaNet as your detector? And could you tell me more about the "Super Planet Recognition" project?



Yes, we are already using RetinaNet. We have achieved 70% accuracy. "Super Planet Recognition" is a competition: "Competitors need to design and train robust models that can detect the difference between the"new map"and the"historical map"and classify the locating parts in order to find suspected supernovae."

**Comment by mehnaz1985** [comment]
@RockyXu66 , why the objects are not detected (BB are not created) in the test phase? another thing what's the purpose of val_df here? Thanks in advance
